jobs/payment-jobs/tasks/cfs_create_invoice_task.py

```diff
@@ -364,80 +349,74 @@ def _active_invoice_reference_subquery(cls):
-                invoice_response = CFSService.create_account_invoice(transaction_number=invoice_number,
-                                                                     line_items=lines,
-                                                                     cfs_account=cfs_account)
-            except Exception as e:  # NOQA # pylint: disable=broad-except
-                # There is a chance that the error is a timeout from CAS side,
-                # so to make sure we are not missing any data, make a GET call for the invoice we tried to create
-                # and use it if it got created.
-                current_app.logger.info(e)  # INFO is intentional as sentry alerted only after the following try/catch
-                has_invoice_created: bool = False
+            current_app.logger.info(
+                f'Found {len(invoices)} EFT invoices for account {payment_account.auth_account_id}')
+            for invoice in invoices:
+                current_app.logger.debug(f'Creating cfs invoice for invoice {invoice.id}')
                 try:
-                    # add a 10 seconds delay here as safe bet, as CFS takes time to create the invoice
-                    time.sleep(10)
-                    invoice_number = generate_transaction_number(str(invoice_number))
-                    invoice_response = CFSService.get_invoice(
-                        cfs_account=cfs_account, inv_number=invoice_number
-                    )
-                    has_invoice_created = invoice_response.get('invoice_number', None) == invoice_number
-                    invoice_total_matches = Decimal(invoice_response.get('total', '0')) == invoice_total
-                except Exception as exc:  # NOQA # pylint: disable=broad-except,unused-variable
-                    # Ignore this error, as it is irrelevant and error on outer level is relevant.
-                    pass
-                # If no invoice is created raise an error for sentry
-                if not has_invoice_created:
-                    capture_message(f'Error on creating EFT invoice: account id={payment_account.id}, '
-                                    f'auth account : {payment_account.auth_account_id}, ERROR : {str(e)}',
-                                    level='error')
-                    current_app.logger.error(e)
-                    continue
-                if not invoice_total_matches:
-                    capture_message(f'Error on creating EFT invoice: account id={payment_account.id}, '
-                                    f'auth account : {payment_account.auth_account_id}, Invoice exists: '
-                                    f' CAS total: {invoice_response.get("total", 0)}, PAY-BC total: {invoice_total}',
-                                    level='error')
-                    current_app.logger.error(e)
-                    continue
+                    invoice_response = CFSService.create_account_invoice(transaction_number=invoice.id,
```

Comment:
Can't do rolled up.. otherwise we can't figure out the line items coming back. PAD and ONLINE banking circumvent this b y issuing CREDIT memos.